src/components/Conference/index.tsx

```diff
@@ -109,9 +110,9 @@ export function Conference({
         </Stack>
         <Stack display='flex' justifyContent='flex-end' minWidth='30%' paddingLeft='30px'>
           <Countdown date={submissionDeadline} renderer={renderCountdown} />
-          <Typography color='#ff6961' fontWeight='bold'>
+          <StyledDeadlineTypography color='#ff6961' fontWeight='bold'>
```

Comment:
There component is already a styled, so there is no need to set inline style again